crates/bitwarden/src/vault/cipher/cipher.rs

```diff
@@ -131,11 +135,15 @@ pub struct CipherListView {
 
 impl KeyEncryptable<Cipher> for CipherView {
     fn encrypt_with_key(self, key: &SymmetricCryptoKey) -> Result<Cipher> {
+        let key_owned = Cipher::get_cipher_key(key, &self.key)?;
+        let key = key_owned.as_ref().unwrap_or(key);
```

Comment:
**suggestion**: I think we can improve the clarify here with some better variable names.

```suggestion
        let ciphers_key = Cipher::get_cipher_key(key, &self.key)?;
        let key = ciphers_key.as_ref().unwrap_or(key);
```